client/scripts/views/components/new_proposal_button.ts

```diff
@@ -12,12 +12,27 @@ import { SubstrateAccount } from 'controllers/chain/substrate/account';
 
 export const getNewProposalMenu = (candidates?: Array<[SubstrateAccount, number]>, mobile?: boolean) => {
   const activeAccount = app.user.activeAccount;
+
+  const topics = app.topics.getByCommunity(app.activeId()).map(({ id, name, featuredInNewPost }) => {
+    return { id, name, featuredInNewPost };
+  }).filter((t) => t.featuredInNewPost).sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
Why are you using both `map` and `filter` here? A reducer could also simplify these lines